docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:stable
+MAINTAINER khmer-project@idyll.org
+
+ENV PACKAGES python-dev zlib1g-dev libbz2-dev gcc git python-setuptools g++ \
+             make ca-certificates python-pip python-wheel python-enchant
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends ${PACKAGES} && \
+    apt-get clean
+
+WORKDIR /home
+
+ARG branch
+ENV branch ${branch:-v2.1.1}
```

Comment:
Just to clarify: the `${branch:-v2.1.1}` syntax defines a default value if otherwise undefined, right?

Reply:
Yep!